minorminer/utils/raster_embedding.py

```diff
@@ -169,7 +169,7 @@ def generator(raster_breadth):
     return raster_breadth
 
 def raster_embedding_search(S, T, timeout=10, raster_breadth=None,
-                            max_num_emb=float('Inf')):
+                            max_num_emb=float('Inf'), tile=None):
```

Comment:
need to update documentation

Reply:
tile: By default the target subgraphs are displaced dwave_networkx chimera, pegasus or zephyr graphs (tiles) parameterized by raster_breadth. A customized tile can replace these default subgraphs. Providing a tile makes raster_breadth redundant.